src/microbe_stage/editor/CellEditorComponent.GUI.cs

```diff
@@ -174,9 +175,42 @@ private void UpdateHitpoints(float hp)
         hpLabel.Value = hp;
     }
 
-    private void UpdateStorage(float storage)
+    private void UpdateStorage(float nominalStorage, Dictionary<Compound, float> storage)
     {
-        storageLabel.Value = (float)Math.Round(storage, 1);
+        storageLabel.Value = nominalStorage;
+
+        var tooltip = ToolTipManager.Instance.GetToolTip("storageDetails", "editor");
```

Comment:
This will print an error if it can't find the tooltip. If tooltipLoaded is allowed to be false, you should use GetToolTipIfExists instead.

Reply:
I see later you basically error out if the tooltip is not found, so no reason to swap to GetToolTipIfExists here instead. As it is an error condition to not find the tooltip.